lib/pure/net.nim

```diff
@@ -478,14 +478,14 @@ proc toSockAddr*(address: IpAddress, port: Port, sa: var Sockaddr_storage,
     sl = sizeof(Sockaddr_in).SockLen
     let s = cast[ptr Sockaddr_in](addr sa)
     s.sin_family = typeof(s.sin_family)(toInt(AF_INET))
-    s.sin_port = port
+    s.sin_port = htons(port)
```

Comment:
See https://www.gta.ufrj.br/ensino/eel878/sockets/sockaddr_inman.html for reference. `fromSockAddrAux` does the opposite using `ntohs`.